android/src/main/java/com/github/alinz/reactnativewebviewbridge/WebViewModule.java

```diff
@@ -0,0 +1,77 @@
+package com.github.alinz.reactnativewebviewbridge;
+
+import com.facebook.react.bridge.ReactApplicationContext;
+import com.facebook.react.bridge.ReactContextBaseJavaModule;
+import com.facebook.react.bridge.ActivityEventListener;
+
+import android.annotation.TargetApi;
+import android.app.Activity;
+import android.content.ClipData;
+import android.content.Intent;
+import android.net.Uri;
+import android.os.Build;
+import android.webkit.ValueCallback;
+import android.util.Log;
+
+import com.facebook.react.common.annotations.VisibleForTesting;
+
+public class WebViewModule extends ReactContextBaseJavaModule implements ActivityEventListener {
+    public ValueCallback<Uri> mUploadMessage;
+    public ValueCallback<Uri[]> mUploadCallbackAboveL;
+
+    @VisibleForTesting
+    public static final String REACT_CLASS = "AndroidWebViewModule";
+
+    public WebViewModule(ReactApplicationContext context){
+        super(context);
+        context.addActivityEventListener(this);
+    }
+
+    @Override
+    public String getName(){
+        return REACT_CLASS;
+    }
+
+    @SuppressWarnings("unused")
+    public Activity getActivity() {
+        return getCurrentActivity();
+    }
+
+    @Override
+    public void onActivityResult(Activity activity, int requestCode, int resultCode, Intent data) {
+        if (requestCode == 1) {
+            if (null == mUploadMessage && null == mUploadCallbackAboveL){
+                return;
+            }
+            Uri result = data == null || resultCode != Activity.RESULT_OK ? null : data.getData();
+            if (mUploadCallbackAboveL != null) {
```

Comment:
why check there if `onActivityResultAboveL` also checks it?
why having `onActivityResultAboveL` at all?